### PR TITLE
updates prerelease notes section

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

File: microsoft-edge/webview2/releasenotes.md [hunk 1]
```diff
@@ -506,7 +506,7 @@ Initial developer preview release.
 [Webview2GettingstartedWpf]: ./gettingstarted/wpf.md "Getting started with WebView2 in WPF | Microsoft Docs"  
 [Webview2HowtoDebug]: ./howto/debug.md "How to debug when developing with WebView2 controls | Microsoft Docs"  
 
-[Webview2ReferenceWin32Icorewebview2experimentalsettingsViewWebview210811GetArebrowseracceleratorkeysenabled]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalsettings?view=webview2-1.0.816&preserve-view=true#get_arebrowseracceleratorkeysenabled "get_AreBrowserAcceleratorKeyPressed - interface ICoreWebView2ExperimentalSettings | Microsoft Docs" 
+[Webview2ReferenceWin32Icorewebview2experimentalsettingsViewWebview210811GetArebrowseracceleratorkeysenabled]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalsettings?view=webview2-1.0.824&preserve-view=true#get_arebrowseracceleratorkeysenabled "get_AreBrowserAcceleratorKeyPressed - interface ICoreWebView2ExperimentalSettings | Microsoft Docs"
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:509]:
```suggestion
[Webview2ReferenceWin32Icorewebview2experimentalsettings2ViewWebview210824GetArebrowseracceleratorkeysenabled]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalsettings2?view=webview2-1.0.824&preserve-view=true#get_arebrowseracceleratorkeysenabled "get_AreBrowserAcceleratorKeyPressed - interface ICoreWebView2ExperimentalSettings | Microsoft Docs" 
```

File: microsoft-edge/webview2/releasenotes.md [hunk 2]
```diff
@@ -14,13 +14,14 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edg
 The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence.  Review the following content for up-to-date information on product announcements, additions, modifications, and breaking changes to the APIs.  
 
 > [!NOTE]
-> Ensure you re-compile your app after updating the NuGet package.  The WebView team recommends using the Canary channel when developing using the prerelease packages, and the evergreen runtime when using released packages.  For more information, navigate to [Versioning][Webview2ConceptsVersioningMatchingWebview2RuntimeVersions].  
+> Ensure you re-compile your app after updating the NuGet package.  The WebView team recommends using the Canary channel when developing using the prerelease packages, and the evergreen runtime when using release packages.  For more information, navigate to [Versioning][Webview2ConceptsVersioningMatchingWebview2RuntimeVersions].  
  
-<!--## 1.0.816-prerelease  
+## 1.0.824-prerelease
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:19]: Could you please update the link for `AreBrowserAcceleratorKeysEnabled` to be `Webview2ReferenceWin32Icorewebview2experimentalsettings2ViewWebview210824GetArebrowseracceleratorkeysenabled`? Somehow I couldn't add the comment to line 31.

**Review by peiche-jessica — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/releasenotes.md (hunk 2 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/releasenotes.md:19]: done!

**Review by peiche-jessica — APPROVED** [review]

**Review by jm-trd-ms — APPROVED** [review]

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**